rust/agama-lib/src/network/types.rs

```diff
@@ -55,6 +55,43 @@ pub enum DeviceType {
     Bridge = 6,
 }
 
+#[derive(Debug, Default, Clone, Copy, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "lowercase")]
```

Comment:
We should use `camelCase`. It won't make a difference in this case, but for consistency...